### PR TITLE
Setup PG, knex migrations, express and make health check route.

### PR DESCRIPTION
**Pull request by Skoryts**

### CODE REVIEW

File: api/app.js [hunk 1]
```diff
@@ -0,0 +1,9 @@
+const express = require('express');
+const app = express();
+const port = 3000;
+
+require('./db');
```
Comment (belevtsev) [inline api/app.js]: You should create a db class and use it everywhere.
Please read how to use pool correctly.

File: api/app.js [hunk 2]
```diff
@@ -0,0 +1,9 @@
+const express = require('express');
+const app = express();
+const port = 3000;
```
Comment (belevtsev) [inline api/app.js]: please use env file

File: api/db.js [hunk 3]
```diff
@@ -0,0 +1,13 @@
+const { Pool } = require('pg');
+const connectionString = 'postgres://wqhrrievdwrbco:4bc4d6ecd015e4a72356e19e9ddaebc8c0c343188716a94914489fd2af1be00c@ec2-54-217-236-201.eu-west-1.compute.amazonaws.com:5432/dfocldlmhfo197';
```
Comment (belevtsev) [inline api/db.js]: please use env file

File: api/db.js [hunk 4]
```diff
@@ -0,0 +1,13 @@
...
+
+const pool = new Pool({
+  connectionString,
+  ssl: true
+});
+
+(async function(){
```
Comment (belevtsev) [inline api/db.js]: is it useful for you? If yes, please tell me why?

File: api/knexfile.js [hunk 5]
```diff
@@ -0,0 +1,35 @@
...
+    pool: {
+      min: 2,
+      max: 10
+    },
+    migrations: {
+      tableName: 'knex_migrations'
+    }
+  },
+
+  production: {
```
Comment (belevtsev) [inline api/knexfile.js]: please remove

File: api/knexfile.js [hunk 6]
```diff
@@ -0,0 +1,35 @@
+module.exports = {
+
+  development: {
+    client: 'postgres',
+    connection: {
```
Comment (belevtsev) [inline api/knexfile.js]: please use env file.

File: api/migrations/20180920142003_test.js [hunk 7]
```diff
@@ -0,0 +1,8 @@
+
+exports.up = function(knex, Promise) {
```
Comment (belevtsev) [inline api/migrations/20180920142003_test.js]: Please create something here.

**Review by belevtsev — CHANGES_REQUESTED** [review]

File: .eslintrc.yml [hunk 8]
```diff
@@ -4,7 +4,6 @@ env:
 
 extends:
   - eslint:recommended
-  - airbnb
```
Comment (belevtsev) [inline .eslintrc.yml:7]: https://www.npmjs.com/package/eslint-config-airbnb-base

File: .eslintrc.yml [hunk 9]
```diff
@@ -16,9 +15,6 @@ rules:
   indent:
     - error
     - 2
-  linebreak-style:
```
Comment (belevtsev) [inline .eslintrc.yml]: linebreak-style should be present.

File: api/config.js [hunk 10]
```diff
@@ -0,0 +1,8 @@
+require('dotenv').config({path: '../.env'});
```
Comment (belevtsev) [inline api/config.js]: please add `{ silent: true }`.

File: api/config.js [hunk 11]
```diff
@@ -0,0 +1,8 @@
...
+
+const config = {
+  port: process.env.PORT,
+  db: process.env.DB_CONNECTION
```
Comment (belevtsev) [inline api/config.js]: `db ` -> `db_uri`

File: api/db.js [hunk 12]
```diff
@@ -0,0 +1,33 @@
...
+class DB {
+  constructor() {
+    this.pool = new Pool({
+      connectionString: config.db,
+      ssl: true
+    });
+  }
+
+  query(query) {
+    return new Promise((resolve, reject) => {
```
Comment (belevtsev) [inline api/db.js]: async await

File: api/migrations/20180926201759_create_users.js [hunk 13]
```diff
@@ -0,0 +1,15 @@
+
+exports.up = function(knex, Promise) {
+  return knex.schema.createTable('users', t => {
+    t.increments('id').unsigned().primary();
```
Comment (belevtsev) [inline api/migrations/20180926201759_create_users.js]: what about uuid?

File: api/migrations/20180926201759_create_users.js [hunk 14]
```diff
@@ -0,0 +1,15 @@
...
+    t.dateTime('createdAt').notNull();
```
Comment (belevtsev) [inline api/migrations/20180926201759_create_users.js]: default value?

File: api/migrations/20180926201759_create_users.js [hunk 15]
```diff
@@ -0,0 +1,15 @@
...
+    t.dateTime('updatedAt').nullable();
```
Comment (belevtsev) [inline api/migrations/20180926201759_create_users.js]: default value?

File: api/migrations/20180926201759_create_users.js [hunk 16]
```diff
@@ -0,0 +1,15 @@
...
+    t.dateTime('deletedAt').nullable();
```
Comment (belevtsev) [inline api/migrations/20180926201759_create_users.js]: default value?

File: .env.example [hunk 17]
```diff
@@ -0,0 +1,2 @@
+PORT=
+DB_CONNECTION=
```
Comment (belevtsev) [inline .env.example]: `DB_CONNECTION ` -> `DB_URI`

**Review by belevtsev — CHANGES_REQUESTED** [review]

File: .eslintrc.yml (hunk 8 above)
Comment (Skoryts) [reply .eslintrc.yml:7]: I have installed, but now I need to left trailing comma! I hate trailing commas.

File: api/migrations/20180926201759_create_users.js (hunk 15 above)
Comment (Skoryts) [reply api/migrations/20180926201759_create_users.js]: Why we need default value? This column we need to fill, when row will be updated.

File: api/migrations/20180926201759_create_users.js (hunk 16 above)
Comment (Skoryts) [reply api/migrations/20180926201759_create_users.js]: Why we need default value? This column we need to fill, when row will be deleted.

File: api/migrations/20180926201759_create_users.js (hunk 15 above)
Comment (belevtsev) [reply api/migrations/20180926201759_create_users.js]: You should have an experience to write triggers.

File: api/migrations/20180926201759_create_users.js (hunk 16 above)
Comment (belevtsev) [reply api/migrations/20180926201759_create_users.js]: You should have an experience to write triggers.

File: .eslintrc.yml (hunk 8 above)
Comment (belevtsev) [reply .eslintrc.yml:7]: let it be.

**Review by belevtsev — CHANGES_REQUESTED** [review]

**Review by belevtsev — APPROVED** [review]

**Pull request merged**